src/components/PartyDropdown.js

```diff
@@ -27,6 +27,7 @@ export default ({ partyId }) => {
   }
   useEffect(() => {
     document.addEventListener("mousedown", handleClickOutside)
+    return () => document.addEventListener("mousedown", handleClickOutside)
```

Comment:
should this line be `document.removeEventListener`?

Reply:
fixed krub